pkg/kubelet/cm/deviceplugin/manager.go

```diff
@@ -524,7 +524,7 @@ func (m *ManagerImpl) devicesToAllocate(podUID, contName, resource string, requi
 	needed := required
 	// Gets list of devices that have already been allocated.
 	// This can happen if a container restarts for example.
```

Comment:
> This can happen if a container restarts for example.

Can this still happen given that it's called from the acceptation predicate ?

Reply:
call from acceptation predicate is not being newly introduced in this PR, therefore, these comments can be corrected independent of this PR.

Reply:
Leaving a TODO if that's the case would be a good idea though.

Reply:
sure